conn/physic/units.go

```diff
@@ -51,12 +51,103 @@ func (a Angle) String() string {
 		i := v / 1000
 		v = v - i*1000
 		return prefix + strconv.FormatInt(int64(i), 10) + "." + prefixZeros(1, int(v)) + "°"
+	case a > (9223372036854775807 - 17453293):
```

Comment:
Can you rebase on top of origin/master so this disappears? Thanks.

But I realized, 17453293 should be Degree instead so then the condition would be:
`case a > maxAngle - Degree:`
which would be much more readable.